pkg/draft/pack/pack_test.go

```diff
@@ -45,24 +48,36 @@ func TestSaveDir(t *testing.T) {
 		t.Errorf("expected there to be no error when writing to %v, got %v", dir, err)
 	}
 
-	_, err = os.Stat(filepath.Join(dir, dockerfileName))
+	fInfo, err := os.Stat(filepath.Join(dir, dockerfileName))
 	if err != nil {
 		if os.IsNotExist(err) {
 			t.Errorf("Expected %s to be created but wasn't", dockerfileName)
 		} else {
 			t.Fatal(err)
 		}
 	}
+	if fInfo.Mode() != dockerPerm && runtime.GOOS != "windows" {
```

Comment:
As there are multiple places in this test where we test for the runtime, would it make sense to use conditional compilation (`// +build windows`)?

Reply:
But as with the previous comment, not necessarily blocking this PR.

Reply:
I've never used conditional compilation before, but that sounds like it would be a really clean solution

Reply:
Hey!
You can find an example with conditional building in tests in this repo - https://github.com/Azure/draft/tree/master/pkg/draft/draftpath

Take a look at that, and if you have any questions I'd be glad to help 😄